ql/ql/src/queries/style/GetAPrimaryQlClassConsistency.ql

```diff
@@ -10,18 +10,31 @@
 
 import ql
 
-from ClassPredicate pred, String constant
+/**
+ * Gets a string that is allowed to be returned from the `getAPrimaryQlClass` implementation
+ * in class `c`.
+ *
+ * For most languages this is just the name of `c`, but QL for Swift implements
+ * the `getAPrimaryQlClass` predicate for a class `Foo` in another class `FooBase`.
+ */
+string getAnAllowedResultForClass(Class c) {
+  result = c.getName()
```

Comment:
I guess it's not possible to restrict this to only apply to `Swift` dbs?

Reply:
We could restrict it to only apply to files that live in the Swift directory, but we decided this solution was sufficient.